src/algorithms/primeFinder.test.js

```diff
@@ -0,0 +1,23 @@
+import isPrime from './primeFinder'
+import {primeFactors} from './primeFinder'
+describe('this is a pseudo TDD build of an isPrime method', () => {
+    test('running isPrime on an even number should return false', () => {
+        let number = 4;
+        expect(isPrime(number)).toBe(false);
+    });
+    test('running isPrime on a prime number should return True', () => {
+        let number = 13;
+        expect(isPrime(number)).toBe(true);
+    });
```

Comment:
I'd recommend testing more than one number.
maybe something like:
```
const primes = [1, 2, 3, 5, 7, 9, 11, 13, 15, 127];
describe('primes should return true',  () => {
  primes.forEach((num) => {
    test(num, () => {
      expect(isPrime(num)).toBe(true);
    });
  });
});
```

Reply:
though you might need to make `num` a string in `test(num, () => {`. I doubt `test`'s first argument is typed (i.e. requires the first argument to be a string), but if it is it could throw an error.